lair_of_xaldern/interface.py

```
@@ -1,33 +1,33 @@
+  go [direction] - Move (North, South, East, West)
   get [item]     - Pick up an item from the room
   drop [item]    - Leave an item behind
   inventory / i  - View your items
   help           - Show these instructions
   quit           - Save and exit the game
-========================================"""
+==================================================="""
     )
 
 
 def show_status(room, inventory, hp, cursor):
```

Ruff (0.14.11)

Unused function argument: `cursor`

(ARG001)